<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movie_info</class>
 <widget class="QDialog" name="movie_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>67</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>POSTER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Movie name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Director</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cast</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>200</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Boxoffice</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_nm">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Movie name</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_cst">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>170</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cast</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_box">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Boxoffice</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_yr">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>100</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_dir">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>140</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Director</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comment">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>comment</string>
   </property>
  </widget>
  <widget class="QFrame" name="commentframe">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPlainTextEdit" name="commentbx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="rateframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="b_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_1">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rate this Movie:-</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="com_ok">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="unameframe">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter user name:-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Rating  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="avg_rating">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
